README.md

```diff
@@ -348,6 +348,7 @@ Below is a list of all the settings that were added in ddnet-insta.
 + `sv_round_stats_discord_webhooks` If set will post score stats there on round end. Can be a comma separated list.
 + `sv_round_stats_http_endpoints` If set will post score stats there on round end. Can be a comma separated list.
 + `sv_round_stats_output_file` If set will write score stats there on round end
++ `sv_freeze_wrong_delay` How many seconds players will remain frozen if a player throws in the wrong spikes for his team (0=off, fng only)
```

Comment:
This still seems to be the old one. Rerunning ``./scripts/gendocs_instagib.sh`` should fix it.